share/lib/python/neuron/rxd/rxd.py

```diff
@@ -630,8 +630,8 @@ def _matrix_to_rxd_sparse(m):
     return (
         n,
         len(nonzero_i),
-        numpy.ascontiguousarray(nonzero_i, dtype=numpy.int_),
-        numpy.ascontiguousarray(nonzero_j, dtype=numpy.int_),
+        numpy.ascontiguousarray(nonzero_i, dtype=ctypes.c_long),
```

Comment:
This function appears to be dead code, as far as I can tell. Changed to `c_long` here as well out of paranoia that it ever gets used again and passed to the C-side, where we use mostly `long`.

Reply:
@ceciliaromaro We should probably try to identify dead code in `rxd` and remove it. The module has a 12-13 year-old code base, so there's probably a nontrivial amount, especially given the major Python to C conversion ~5 years ago.